packages/manager/modules/pci/src/projects/project/storages/snapshots/snapshot/delete/delete.html

```diff
@@ -0,0 +1,20 @@
+<oui-modal
+    data-heading="{{ 'pci_projects_project_storages_snapshots_snapshot_delete_title' | translate }}"
+    data-primary-action="$ctrl.deleteSnapshot()"
+    data-primary-label="{{:: 'pci_projects_project_storages_snapshots_snapshot_delete_submit_label' | translate }}"
+    data-primary-disabled="$ctrl.loadings.init || $ctrl.loadings.save"
+    data-secondary-action="$ctrl.goBack()"
+    data-secondary-label="{{:: 'pci_projects_project_storages_snapshots_snapshot_delete_cancel_label' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+    data-loading="$ctrl.loadings.init">
+
+    <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+    <p data-translate="pci_projects_project_storages_snapshots_snapshot_delete_content"
+        data-translate-values="{ snapshot: $ctrl.snapshot.name }"></p>
+
+    <div data-ng-if="$ctrl.loadings.save" class="text-center">
```

Comment:
Couldn't be shared with the `loading` attribut from the modal component instead?